src/cmdstan-guide/command_line_options.Rmd

```diff
@@ -77,6 +77,10 @@ The default value is 6. &nbsp;
 ***Note:*** increasing `sig_figs` above the default will increase the size of
```

Comment:
We aren't respecting the `sig_figs` argument are we? But this would be a problem we need to address in cmdstan I guess, not here.

Reply:
We do not. But I guess we should. Will fix that.